application/src/main/java/org/togetherjava/tjbot/commands/system/BotCore.java

```diff
@@ -15,6 +15,8 @@
 import net.dv8tion.jda.api.interactions.commands.Command;
 import net.dv8tion.jda.api.interactions.components.ComponentInteraction;
 import net.dv8tion.jda.api.requests.ErrorResponse;
+import org.jetbrains.annotations.NotNull;
```

Comment:
use Nonnull (which is the default now anyways)

Reply:
has  
![image](https://user-images.githubusercontent.com/49957334/193246145-a675d9d8-976a-416f-ba46-da67579686af.png)
changed.